app/controllers/api_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ApiController < ApplicationController
+  def return_posts
+    @posts = Post.where(author_id: params[:user_id])
+    render json: @posts
+  end
+
+  def return_comments
+    @comments = Comment.where(post_id: params[:post_id])
+    render json: @comments
+  end
+
+  def create_comment
+    text = params[:text]
+    post_id = params[:post_id]
+    comment = Comment.create(text:, author: current_user, post_id:)
+    render json: comment, status: 200
+  end
```

Comment:
- [ ] when I tried to create a comment using the api and I was signed in  it returns the following error so please try to make sure to fix that 
<img width="559" alt="Screenshot 2023-02-03 at 16 37 41" src="https://user-images.githubusercontent.com/61790587/216644108-78d3d43a-21f5-4033-bb79-199d0d010ae2.png">

Reply:
@youmari Hi, could you tell me how you tried to add the comment?

Reply:
@laiifuu When I added from the browser like that `http://127.0.0.1:3000/users/1/api/v1/posts/1/comments/toto`it worked but if I use a app for apis like postman or thunder it didn't work and the api should work anywhere

Reply:
Okay, thanks

Reply:
@laiifuu welcome and it should be `post` method not a `get`